### ISSUE TITLE
Should be able to click on a visible element

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#554]
### Expected behavior
Should be able to click on a visible element

### Actual behavior
 ✘ Error: Element with text Add to cart not found, run `.trace` for more info.

### Steps to reproduce
* In the REPL follow the following steps
```
> openBrowser()
 ✔ Browser opened
> goto("http://automationpractice.com/index.php")
 ✔ Navigated to url http://automationpractice.com/index.php
> hover("Printed Dress")
 ✔ Hovered over the element matching text "Printed Dress"
> click("Quick view")
 ✔ Clicked element matching text "Quick view"
> click("Add to cart")
 ✘ Error: Element with text Add to cart not found, run `.trace` for more info.
```
**Note:** The element is a button available in the popup with text inside a span
```
<button type="submit" name="Submit" class="exclusive">
	<span>Add to cart</span>
</button>
```
```
### Versions
```
commit: 3bb2ade397840ad2326026bb7fad840161e673ea
Version: 0.8.0 (Chromium: 74.0.3723.0)
```

**Comment by NivedhaSenthil** [comment]
Fix available in f6e741f

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the commit version ``` a622aa5c84bc8544ef3e3028ae73afcb55a96b1b ```
